Tests/UpstreamIntegrationTests/UpstreamIntegrationTests.swift

```diff
@@ -17,29 +17,25 @@ import AEPServices
 import Foundation
 import XCTest
 
-/// This test class validates proper intergration with upstream services, specifically Edge Network
+/// Performs validation on intergration with the Edge Network upstream service
 class UpstreamIntegrationTests: TestBase {
     private var edgeEnvironment: EdgeEnvironment = .prod
     private var edgeLocationHint: EdgeLocationHint?
-    
+
     private var networkService: RealNetworkService = RealNetworkService()
 
     let LOG_SOURCE = "UpstreamIntegrationTests"
 
     let asyncTimeout: TimeInterval = 10
 
-    // Run once per test suite
-    override class func setUp() {
-        super.setUp()
-
-        TestBase.debugEnabled = true
-
-    }
-
     // Run before each test case
     override func setUp() {
+        ServiceProvider.shared.networkService = networkService
+
         super.setUp()
+
         continueAfterFailure = true
+        TestBase.debugEnabled = true
         // Extract Edge Network environment level from shell environment; see init for default value
         self.edgeEnvironment = EdgeEnvironment()
         print("Using Edge Network environment: \(edgeEnvironment.rawValue)")
```

Comment:
@timkimadobe I anticipate some tests will be failing if we set the a location hint as input for the test run, as we are not taking it into account when setting the expected NetworkRequest?